meterfs/test_meterfs_allocate.c

```diff
@@ -95,10 +98,10 @@ TEST(meterfs_allocate, var_init_args)
 TEST_GROUP_RUNNER(meterfs_allocate)
 {
 	RUN_TEST_CASE(meterfs_allocate, big_file);
-	RUN_TEST_CASE(meterfs_allocate, many_files);
+	/* RUN_TEST_CASE(meterfs_allocate, many_files);
```

Comment:
Well, disabling tests is not exactly fixing them ;)

Reply:
Forgot to reenable it, my bad